src/tools/wasm-interp.cc

```diff
@@ -327,6 +465,12 @@ int ProgramMain(int argc, char** argv) {
   s_store.setFeatures(s_features);
 
   wabt::Result result = ReadAndRunModule(s_infile);
+
+  for (auto& pair : s_registry) {
+    pair.second.clear();
+  }
+  s_registry.clear();
```

Comment:
I think it should be enough to just call `.clear()` on the `s_registry` .. this will automatically destroy the `ExportMap` object which are the values in the map, no?